fhttp/http_client.go

```diff
@@ -426,6 +434,16 @@ func NewStdClient(o *HTTPOptions) (*Client, error) {
 			if o.Resolve != "" {
 				addr = o.Resolve + addr[strings.LastIndex(addr, ":"):]
 			}
+			// TODO: Find out how many time this get called. Should be num of conn + error
```

Comment:
do increment numsockets while at it?

Reply:
added implementation and a unit test. Now the stdclient will also show the socket count.

Reply:
awesome!